examples/pubsub/tutorial_pubsub_configuration.c

```diff
@@ -0,0 +1,180 @@
+/* This work is licensed under a Creative Commons CCZero 1.0 Universal License.
+ * See http://creativecommons.org/publicdomain/zero/1.0/ for more information. */
```

Comment:
Maybe it would be better to name the file server_pubsub_file_configuration.c. There is no clear boundary between tutorials and other examples but for this non-standard behavior we should avoid the 'tutorial' from my point of view.